src/routes/form-title.tsx

```diff
@@ -18,11 +18,12 @@ const TitleContainer = styled.div`
 `;
 
 const Title = styled(Innholdstittel)`
-    @media ${device.mobileS} {
```

Comment:
Trenger ikke spesifisere to media-queries. Default er `0` og for store mobiler og oppover er det `8px`.